node_modules/app/format_date.js

```diff
@@ -6,12 +6,16 @@ module.exports = function (d) {
   return date[1]
 }
 
+module.exports.date_string = function (d) {
```

Comment:
Haha. Okay will do this, get rid of that pesky repetition. Also I was going to clean up payments on live DB but then saw that the majority of the ones created today were not me. I'll just leave it?